README.md

```diff
@@ -29,7 +31,9 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The rails router fires off methods defined in the controller based on the http request and type received.
```

Comment:
Sort of here. the router actually determines which controller to send the request to, and what method should be taken